lib/pcb/create-svg-objects-from-pcb-rats-nest/get-element-position.ts

```diff
@@ -0,0 +1,19 @@
+import type { AnyCircuitElement } from "circuit-json"
+
+export const getElementPosition = (
+  id: string,
+  soup: AnyCircuitElement[],
+): { x: number; y: number } | null => {
+  const elm = soup.find((element) => {
+    const elmId =
+      (element as any).id ||
+      (element as any).pcb_port_id ||
+      (element as any).source_port_id
+    return elmId === id
+  })
+
+  if (elm && "x" in elm && "y" in elm) {
+    return { x: (elm as any).x, y: (elm as any).y }
+  }
+  return null
+}
```

Comment:
do you still need this? It's very hacky and won't work correctly because it doesn't filter based on elm.type